airbyte-ci/connectors/metadata_service/lib/metadata_service/gcs_upload.py

```diff
@@ -121,6 +136,25 @@ def _write_metadata_to_tmp_file(metadata_dict: dict) -> Path:
 # 🛠️ HELPERS
 
 
+def _delete_blob_from_gcs(blob_to_delete: storage.blob.Blob) -> bool:
```

Comment:
Looks like this isn't being used. If we do use it, let's get rid of the return value unless it's definitely needed.